Discord Server.url

```diff
@@ -1,2 +1,2 @@
 [InternetShortcut]
-URL=https://discordapp.com/invite/015OTigbS9wyJlBDP
+URL=https://discord.gg/015OTigbS9wyJlBDP
```

Comment:
```suggestion
URL=https://discordapp.com/invite/015OTigbS9wyJlBDP
```

Reply:
RS did say he doesn't like the idea of redirected URLs, so, let's stay where we were.